arcgis-ios-sdk-samples/Geometry/Buffer list/BufferListViewController.swift

```diff
@@ -0,0 +1,200 @@
+// Copyright 2020 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class BufferListViewController: UIViewController {
+    // MARK: Storyboard views
+    
+    /// The create button.
+    @IBOutlet weak var createBarButtonItem: UIBarButtonItem!
+    /// The clear button.
+    @IBOutlet weak var clearBarButtonItem: UIBarButtonItem!
+    /// A label to display status message.
+    @IBOutlet weak var statusLabel: UILabel!
+    /// A switch to enable "union" for buffer operation.
+    @IBOutlet weak var isUnionSwitch: UISwitch!
+    /// The map view managed by the view controller.
+    @IBOutlet weak var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap(imageLayer: mapImageLayer)
+            mapView.graphicsOverlays.addObjects(from: [boundaryGraphicsOverlay, bufferGraphicsOverlay, tappedLocationsGraphicsOverlay])
+            mapView.touchDelegate = self
+        }
+    }
+    
+    // MARK: Instance properties
+    
+    /// An image layer as the base layer of the map.
+    let mapImageLayer = AGSArcGISMapImageLayer(url: URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/USA/MapServer")!)
+    /// A polygon that represents the valid area of use for the spatial reference.
+    let boundaryPolygon: AGSPolygon = {
+        let boundaryPoints = [
+            AGSPoint(x: -103.070, y: 31.720, spatialReference: .wgs84()),
+            AGSPoint(x: -103.070, y: 34.580, spatialReference: .wgs84()),
+            AGSPoint(x: -94.000, y: 34.580, spatialReference: .wgs84()),
+            AGSPoint(x: -94.00, y: 31.720, spatialReference: .wgs84())
+        ]
+        let polygon = AGSPolygon(points: boundaryPoints)
+        let statePlaneNorthCentralTexas = AGSSpatialReference(wkid: 32038)!
+        return AGSGeometryEngine.projectGeometry(polygon, to: statePlaneNorthCentralTexas) as! AGSPolygon
+    }()
+    
+    /// An overlay to display the boundary.
+    var boundaryGraphicsOverlay: AGSGraphicsOverlay {
+        let overlay = AGSGraphicsOverlay()
+        let lineSymbol = AGSSimpleLineSymbol(style: .dash, color: .red, width: 5)
+        let boundaryGraphic = AGSGraphic(geometry: boundaryPolygon, symbol: lineSymbol)
+        overlay.graphics.add(boundaryGraphic)
+        return overlay
+    }
```

Comment:
Since this overlay depends on the geometry of the `boundaryPolygon`, it is better to be instantiated by an instance method. However, I found it looking more "symmetric" to also create it as a computed variable. Wait for Phil's thought.